src/PhpBrew/VariantBuilder.php

```diff
@@ -269,6 +269,24 @@ public function __construct()
         };
 
         $this->variants['pcre'] = function (ConfigureParameters $params, Build $build, $value) {
+            // Apple Silicon will crash on 8.0 due to bug in bundle PCRE2 so get an updated version if we can
+            // PHP 8.1 and above already have the fix bundled
+            if ($build->compareVersion('8.0') >= 0 && $build->compareVersion('8.1') < 0 && $build->osName === 'Darwin' && $build->osArch === 'arm64') {
```

Comment:
Shouldn't this check for the exact version fixed?

```suggestion
            if ($build->compareVersion('8.0') >= 0 && $build->compareVersion('8.1.11') < 0 && $build->osName === 'Darwin' && $build->osArch === 'arm64') {
```